showcase/src/app/app.module.ts

```diff
@@ -235,7 +235,9 @@ export const metaReducers: MetaReducer<State>[] = ENV !== "production" ? [logger
 			position: "top right",
 			actionClasses: []
 		}),
-		StarkSessionUiModule.forRoot(),
+		StarkSessionUiModule.forRoot({
+			enableTimeoutWarningDialogEffects: true
```

Comment:
You can revert this change when you rename this option to "timeoutWarningDialogDisabled" as I mentioned in my comments above ;)